CHANGELOG.md

```diff
@@ -9,7 +9,9 @@
 - [core] feat: Export `makeMain` (#2665)
 - [core] fix: Call `bindClient` when creating new `Hub` to make integrations work automatically (#2665)
 - [gatsby] feat: Add @sentry/gatsby package (#2652)
-- [core] ref: Rename `whitelistUrls/blacklistUrls` to `allowUrls/denyUrls`
+- [apm] feat: Add `scope.getTransaction` to return a Transaction if it exists (#2668)
```

Comment:
This to conflicts with the PR description -- PR description says it takes a callback, here it says it returns a Transaction.